src/main/java/com/app/edu/config/DataLoader.java

```diff
@@ -193,10 +193,18 @@ CommandLineRunner populateData() {
             ));
 
             // shapes
-            shapeRepository.save(new ShapeEntity(1, "Cerc","src/main/resources/static/sounds/shapesSounds/circleSound.wav",
+            shapeRepository.save(new ShapeEntity(1, "Cerc", "src/main/resources/static/sounds/shapesSounds/circleSound.wav",
```

Comment:
please change label to human language

Reply:
please don't make reviews after 6pm

Reply:
stfu

Reply:
prod urgent feature

Reply:
![image](https://github.com/Midn18/edu-ciupi/assets/66823804/6ef610db-10ea-42bd-bd0d-22b5f021ae8e)

Reply:
moare pradactionu fara poze cu null pointer

Reply:
gata cu fotosinteza, toata lumea la culcare